src/internet/subsonic/subsonicservice.cpp

```diff
@@ -494,6 +495,15 @@ void SubsonicLibraryScanner::OnGetAlbumFinished(QNetworkReply* reply) {
   }
 }
 
+void SubsonicLibraryScanner::OnGetAlbumCoverFinished(QNetworkReply* reply, QString albumid) {
+  reply->deleteLater();
+  pending_requests_.remove(reply);
+
+  QByteArray image = reply->readAll();
+
+  service_->emitImageLoaded(albumid, QImage::fromData(image));
```

Comment:
Yeah I knew you'd say that, but again, hack over hack.
Do you guys plan to rework the cover art links? We could create URI handlers so services can hook them, more generic and abstract. What do you think?